I don't understand the Docs CI failure:

```
PATH=./venv/bin:$PATH sphinx-build -b doctest -d build/doctrees  -q -W --keep-going -j4 -W . build/doctest 
...
make[1]: *** [Makefile:51: build] Error 1
make[1]: Leaving directory '/home/runner/work/cpython/cpython/Doc'
Testing of doctests in the sources finished, look at the results in build/doctest/output.txt
make: *** [Makefile:131: doctest] Error 1
make: Leaving directory '/home/runner/work/cpython/cpython/Doc'
Error: Process completed with exit code 2.
```

--- a/Misc/NEWS.d/next/Library/2022-01-26-23-58-48.bpo-45162.4Jmg_j.rst
+++ b/Misc/NEWS.d/next/Library/2022-01-26-23-58-48.bpo-45162.4Jmg_j.rst
@@ -1,3 +1,3 @@
-The deprecated :module:`unittest` APIs removed in 3.11a1 have been
+The deprecated :mod:`unittest` APIs removed in 3.11a1 have been
 temporarily restored to be removed in 3.12 while cleanups in external
 projects go in.

